packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -135,7 +143,7 @@ contract Initializer_Test is Bridge_Initializer {
         command[1] = "-c";
         command[2] = string.concat(
             Executables.find,
-            " src/L1 -type f -exec basename {} \\;",
+            " src/L1 src/L2 -type f -exec basename {} \\;",
```

Comment:
should rename this function since it looks at both L1 and L2 contracts.

Reply:
did this, good catch